x-pack/plugins/security_solution/public/timelines/components/timeline/header/__snapshots__/index.test.tsx.snap

```diff
@@ -21,14 +21,12 @@ exports[`Header rendering renders correctly against snapshot 1`] = `
           "get": [MockFunction],
           "get$": [MockFunction],
           "getAll": [MockFunction],
-          "getSaved$": [MockFunction],
```

Comment:
Please, consider fixing the test. It shouldn't fail whenever the Core team updates any of the public contract methods. It creates a lot of false-negative failures.